cmd/server/main.go

```diff
@@ -111,17 +116,31 @@ func main() {
 		cfg.NetworkName = networkName
 	}
 
-	network := &types.NetworkIdentifier{
+	networkP := &types.NetworkIdentifier{
+		Blockchain: service.BlockchainName,
+		Network:    cfg.NetworkName,
+		SubNetworkIdentifier: &types.SubNetworkIdentifier{
+			Network: mapper.PChainNetworkIdentifier,
+		},
+	}
+	networkC := &types.NetworkIdentifier{
```

Comment:
should probably be:
```
networkC := &types.NetworkIdentifier{
	Blockchain: service.BlockchainName,
	Network:    cfg.NetworkName,
	SubNetworkIdentifier: &types.SubNetworkIdentifier{
		Network: mapper.CChainNetworkIdentifier,
	},
}
```

Reply:
SubNetworkIdentifier field for C-chain was omitted deliberately for backwards compatibility with existing consumers of Rosetta who makes requests without specifying the SubNetworkIdentifier field for C-chain. IIRC, suggestion came from @patrick-ogrady.

Reply:
Thanks for the context, @bilgehansahin